### DISCUSSION TITLE
guard config does not work. v2.

### DISCUSSION THREAD

**Discussion by magic-thomas** [filamentphp/filament#8391] [Help]
Hi . 

I am using v2.

in config/filament.php, there is an option .

```php
'auth' => [
        'guard' => env('FILAMENT_AUTH_GUARD', 'web'), 
        'pages' => [
            'login' => \Filament\Http\Livewire\Auth\Login::class,
        ],
    ],
```

When I set guard from 'web' to 'auth'  , it shows error 

```
Auth guard [auth] is not defined.
```

And even custom middleware also throw same error.

How can I change guard ?

**Comment by danharrin** [comment]
Sounds like you havent defined the guard in `config/auth.php`
